### PR TITLE
Hotfix/ios orlib

### PR DESCRIPTION
**Pull request by Miggets7**

### CODE REVIEW

File: console/android/GenericApp/app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -1,6 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="io.openremote.app">
+<manifest xmlns:android="http://schemas.android.com/apk/res/android">
```
Comment (richturner) [inline console/android/GenericApp/app/src/main/AndroidManifest.xml:2]: Should package attribute be removed?

Comment (Miggets7) [reply console/android/GenericApp/app/src/main/AndroidManifest.xml:2]: Yes, in newer android grade plugin, this is specified elsewhere

**Comment by ebariaux** [comment]
Ok for me to merge, just did a review from the diff in GitHub, pretty difficult to understand the changes from the Xcode project file.
@Miggets7 can you just describe quickly what you did, I'll re-merge master in my branch after you merge this and I'm afraid I'll get conflict so I might as well manually re-apply some steps.

**Review by ebariaux — APPROVED** [review]
OK for iOS, haven't looked at Android

**Pull request merged**